MAVProxy/modules/mavproxy_paramedit/param_editor.py

```diff
@@ -85,6 +85,7 @@ class ParamEditorMain(object):
     def __init__(self, mpstate):
         self.param_received = {}
         self.paramchanged = {}
+        self.fltmode_rc = 1100
```

Comment:
I think it should default to None, and check for None in the code that uses it

Reply:
Corrected